### PR TITLE
2019-11-04 TSC meeting notes

### PR DESCRIPTION
**Pull request by michdolan**
Signed-off-by: michdolan <michdolan@gmail.com>

### CODE REVIEW

File: docs/tsc/meetings/2019-11-04.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+  * [X] Mark Boorer (_TSC_) - Industrial Light & Magic
+  * [X] Sean Cooper (_TSC_) - DNEG
+  * [X] Michael Dolan (_TSC Chair_) - Sony Pictures Imageworks
+  * [X] Larry Gritz (_TSC_) - Sony Pictures Imageworks
+  * [X] Patrick Hodoul (_TSC_) - Autodesk
+  * [X] John Mertic - Academy Software Foundation / Linux Foundation
+  * [ ] Carl Rand (_TSC_) - Foundry
+  * [X] Doug Walker (_TSC Chief Architect_) - Autodesk
+  * [X] Kevin Wheatley (_TSC_) - Framestore
+
```
Comment (hodoulp) [inline docs/tsc/meetings/2019-11-04.md]: * [X] Bernard Lefebvre - Autodesk

File: docs/tsc/meetings/2019-11-04.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+        - **TODO**: Patrick will investigate numerical ids.
+    - Could OLDYAML support be definitively removed?
+        - All agree, yes. Remove support for old yaml-cpp version.
+
+* PR #875, active_views filtering and ordering
+    - PR submitted by Patrick, and has been approved for merge.
+    - Displays and views are now filtered and ordered similarly, but there's 
+      still no validation that an active display or view is actually defined.
+    - Patrick: View behavior was modified to exactly match display behavior.
+    - Doug: Needs second pass at some point to impreove error handling, but
```
Comment (hodoulp) [inline docs/tsc/meetings/2019-11-04.md]: typo 'improeve' -> 'improve'

File: docs/tsc/meetings/2019-11-04.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+      apps have different needs. Could it be unified around what an application 
+      is trying to do?
+        - Doug: More specific roles would be good.
+    - **TODO**: Improve documentation around standard roles and how they are 
+      intended to be used.
+
+- Patrick: Question about macOS CI with LFRE
+    - Highlights our lack of control of our build machines
+    - It would be good to know what we are compiling on, especially if we 
+      someday distribute build artifact binaries. Need to know mahcine,
```
Comment (hodoulp) [inline docs/tsc/meetings/2019-11-04.md]: typo 'mahcine' -> 'machine'

File: docs/tsc/meetings/2019-11-04.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+* PR #875, active_views filtering and ordering
+    - PR submitted by Patrick, and has been approved for merge.
+    - Displays and views are now filtered and ordered similarly, but there's 
+      still no validation that an active display or view is actually defined.
+    - Patrick: View behavior was modified to exactly match display behavior.
+    - Doug: Needs second pass at some point to impreove error handling, but 
+      nervous to change long standing behavior.
+        - If an active display doesnt exist, should we throw exception? It 
+          currently doesn't.
+        - Parick: active_views applies to all displays, though views can differ
```
Comment (hodoulp) [inline docs/tsc/meetings/2019-11-04.md]: typo -> 'Parick' -> 'Patrick'

Comment (michdolan) [reply docs/tsc/meetings/2019-11-04.md]: Fixed.

File: docs/tsc/meetings/2019-11-04.md (hunk 3 above)
Comment (michdolan) [reply docs/tsc/meetings/2019-11-04.md]: Fixed.

File: docs/tsc/meetings/2019-11-04.md (hunk 2 above)
Comment (michdolan) [reply docs/tsc/meetings/2019-11-04.md]: Fixed.

File: docs/tsc/meetings/2019-11-04.md (hunk 1 above)
Comment (michdolan) [reply docs/tsc/meetings/2019-11-04.md]: Fixed.

**Pull request merged**